app/components/Form/ApplicationFormStatus.tsx

```diff
@@ -88,7 +88,9 @@ const ApplicationFormStatus: React.FC<Props> = ({
         <AppStatus>{status}</AppStatus>
         <AppName>{formData?.projectInformation?.projectTitle}</AppName>
       </StatusNameFlex>
-      <SavingStatusContainer>{savingStatusIndicator}</SavingStatusContainer>
+      <SavingStatusContainer aria-live="off" aria-label="status container">
```

Comment:
```suggestion
      <SavingStatusContainer aria-live="off" aria-label="form saving status">
```

This label would be read by a screen reader, so it should make sense to our user